### ISSUE TITLE
Add option to align icon/text vertically

### ISSUE THREAD

**Issue by chronus7** [dunst-project/dunst#486]
In it's current state (v1.3.0), dunst seems to vertically center the icon and text. I personally find this irritating and would prefer to align them to the top of the notification.

The following graphics demonstrate/clarify the idea with a) an icon smaller than the text and b) a much larger icon than the text.

```
Center alignment (current behaviour)
┌───────────────────┐  |  ┌───────────────────┐
│      first line   │  |  │ ####              │
│  #   second       │  |  │ ####  one line    │
│      third line   │  |  │ ####              │
└───────────────────┘  |  └───────────────────┘

Top alignment
┌───────────────────┐  |  ┌───────────────────┐
│  #   first line   │  |  │ ####  one line    │
│      second       │  |  │ ####              │
│      third line   │  |  │ ####              │
└───────────────────┘  |  └───────────────────┘

Bottom alignment
┌───────────────────┐  |  ┌───────────────────┐
│      first line   │  |  │ ####              │
│      second       │  |  │ ####              │
│  #   third line   │  |  │ ####  one line    │
└───────────────────┘  |  └───────────────────┘
```

Would it be possible to implement such alignment? I would much appreciate it.

**Comment by FeintGS** [comment]
Agreed, I can totally understand why people would be driven nuts by a limitation in the current limitation like this. For future readers who are looking for a solution to this problem, here's a workaround hack you can use. Basically, what you do is trick dunst into preserving the leading/trailing whitespace in your string by using a "pseudo" space character. This way, you can vertically top-align or bottom-align your notification text.



**Before the hack:**
```
notify-send "🎵 ~Now playing~ 🎵" "$SONG_TITLE\n$ALBUM_NAME"
```
![before](https://user-images.githubusercontent.com/40724862/56878532-9273b680-6a22-11e9-9e16-11df0a5c0c25.png)

**After the hack:**
```
# NOTE!!! The "space" at the end is actually the Unicode symbol '\u0020'
notify-send "🎵 ~Now playing~ 🎵" "$SONG_TITLE\n$ALBUM_NAME\n\n\n "
```
![after](https://user-images.githubusercontent.com/40724862/56878537-97386a80-6a22-11e9-8830-a9a82dfbb8f3.png)

Good luck! [Here's a video demo of it in action](https://twitter.com/FeintGS/status/1122726445552300033)

**Comment by kushraj** [comment]
Hi, is there any possibility this will be taken up any time soon ?
been really looking forward to this since a while..

**Comment by tsipinakis** [comment]
Unfortunately, I do not have enough time to dedicate to proper dunst development for the time being other than bugfixing. However if anyone wants to step up to implement this I'd gladly accept a PR, and I'm obviously still active here if there are any questions.

Most of the positioning work is done in [render_content](https://github.com/dunst-project/dunst/blob/master/src/draw.c#L508) so if anyone wants to implement it that's the place to start.

**Comment by chronus7** [comment]
> Most of the positioning work is done in [render_content](https://github.com/dunst-project/dunst/blob/master/src/draw.c#L508) so if anyone wants to implement it that's the place to start.

Uh, this looks intriguing. I hacked at it shortly and it looks promising. Will try to have another shot at it on the weekend. Thanks for the pointer :)

**Comment by tsipinakis** [comment]
Implemented with #684!
